cartographer/sensor/data.h

```diff
@@ -41,6 +41,10 @@ struct Data {
     PointCloud ranges;
   };
 
+  struct FixedFramePose {
```

Comment:
Why not directly use a `transform::Rigid3d` similar to the `odometry_pose`? Seems this struct is not needed.